### PR TITLE
Add support for high DPI to the Burn engine

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: src/burn/stub/stub_arm.manifest [hunk 1]
```diff
@@ -1,16 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <!--
-  <copyright file="stub_arm.manifest" company="Outercurve Foundation">
+  <copyright file="stub.manifest" company="Outercurve Foundation">
```
Comment (robmen) [inline src/burn/stub/stub_arm.manifest]: This doesn't look right. This is the ARM stub manifest, right?

File: src/burn/stub/stub_arm.manifest [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
     Copyright (c) 2004, Outercurve Foundation.
     This software is released under Microsoft Reciprocal License (MS-RL).
     The license and further copyright text can be found in the file
     LICENSE.TXT at the root directory of the distribution.
   </copyright>
 -->
-<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
- <assemblyIdentity name="setup.exe" version="1.0.0.0" processorArchitecture="arm" type="win32"/>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3" manifestVersion="1.0">
+ <assemblyIdentity name="setup.exe" version="1.0.0.0" processorArchitecture="x86" type="win32"/>
```
Comment (robmen) [inline src/burn/stub/stub_arm.manifest]: This too seems wrong if this is the ARM stub manifest.

File: src/burn/stub/stub_arm.manifest (hunk 1 above)
Comment (heaths) [inline src/burn/stub/stub_arm.manifest]: My bad. Copy/paste mistake. New commit coming...

**Comment by heaths** [comment]
BTW, you saw the note about _wixstdba_ in http://wixtoolset.org/issues/4541/?

**Comment by robmen** [comment]
There is already bug open on high-dpi issues in wixstdba.

**Comment by robmen** [comment]
Thanks for fixing the arm stuff so quickly.

**Pull request merged**